### ISSUE TITLE
Language button

### ISSUE THREAD

**Issue by dennmuel** [UB-Mannheim/PalMA#98]
Should we add a button to change the language or do we want to keep it conditional to the browser language?

**Comment by wagneral** [comment]
Think we should add an additional button if it does not take a high effort. 

Users don't necessary know what they have (installed) on their own or on lent devices. Furthermore language buttons are common for websites.

**Comment by stweil** [comment]
The current implementation has two issues:

- When I press the language button and select a language, the user interface switches to a different view. This is an unexpected behavior.
- The language selection is currently not persistent. That limits its value.

**Comment by dennmuel** [comment]
Yes, the language selection causes the page to reload. It reloads to the control tab by default, which is an intended behaviour (makes sense e.g. after adding a content to be lead directly to where you can control said content). We could see how we could make the site reload to the section where the language selection is.

I am not sure what you mean by "not persistent", but I guess you mean, that the languages selectable are not all translated yet. This can be solved either by translating everything (long term solution) or by deleting the directories of untranslated languages (short term solution).

Language selection during login sounds like a good idea. Or does anyone else see any problem with this?

**Comment by stweil** [comment]
"Persistent" means that the chosen language should be saved as a cookie in the browser and used the next time(s) when PalMA is used again. Otherwise a user would have to select the language each time when (s)he uses PalMA.
